pkg/alicloud/apis/alicloud_provider_spec.go

```diff
@@ -29,13 +31,13 @@ type ProviderSpec struct {
 	DataDisks               []AlicloudDataDisk      `json:"dataDisks,omitempty"`
 	InstanceChargeType      string                  `json:"instanceChargeType,omitempty"`
 	InternetChargeType      string                  `json:"internetChargeType,omitempty"`
-	InternetMaxBandwidthIn  *int32                  `json:"internetMaxBandwidthIn,omitempty"`
-	InternetMaxBandwidthOut *int32                  `json:"internetMaxBandwidthOut,omitempty"`
+	InternetMaxBandwidthIn  *int                    `json:"internetMaxBandwidthIn,omitempty"`
```

Comment:
I hope changing these fields from `int32` to `int` shouldn't be an issue. Doesn't seem to be, just double confirming.

Reply:
because these two variables are defined as `*int`, see https://github.com/gardener/machine-controller-manager/blob/master/pkg/apis/machine/v1alpha1/alicoud_machineclass_types.go#L87-L88, in order to make the migration more consistent, I made this change.

Reply:
Okay thanks for the explaination.